<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Special Effect Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="shadowTab">
      <attribute name="title">
       <string>Shadow</string>
      </attribute>
      <layout class="QFormLayout" name="layShadowTab">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="shadow">
         <property name="text">
          <string>Enable client-side shadows on windows</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="shadow_radius_label">
         <property name="text">
          <string>The blur radius for shadows</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="shadow_radius"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="shadow_offset_x_label">
         <property name="text">
          <string>The left offset for shadows</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="shadow_offset_x">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="shadow_offset_y_label">
         <property name="text">
          <string>The top offset for shadows</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="shadow_offset_y">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="shadow_opacity_label">
         <property name="text">
          <string>The translucency for shadows</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="shadow_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="shadow_color_label">
         <property name="text">
          <string>Color of shadow</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="shadow_color">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="xinerama_shadow_crop">
         <property name="text">
          <string>Crop shadows of maximized windows from extended displays</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="opacityTab">
      <attribute name="title">
       <string>Opacity</string>
      </attribute>
      <layout class="QFormLayout" name="layOpacityTab">
       <item row="0" column="0">
        <widget class="QLabel" name="inactive_opacity_label">
         <property name="text">
          <string>Default opacity of inactive windows</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="inactive_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="active_opacity_label">
         <property name="text">
          <string>Default opacity for active windows</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="active_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="frame_opacity_label">
         <property name="text">
          <string>Opacity of window titlebars and borders</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="frame_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="inactive_dim_label">
         <property name="text">
          <string>Dim inactive windows</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="inactive_dim">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="inactive_dim_fixed">
         <property name="text">
          <string>Do not let dimness adjust based on window opacity</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background">
         <property name="text">
          <string>Blur background of transparent windows</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background_frame">
         <property name="text">
          <string>Blur background of opaque windows with transparent frames as well</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background_fixed">
         <property name="text">
          <string>Do not let blur radius adjust based on window opacity</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fadeTab">
      <attribute name="title">
       <string>Fade</string>
      </attribute>
      <layout class="QFormLayout" name="layFadeTab">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="fading">
         <property name="text">
          <string>Fade windows during opacity changes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="fade_delta_label">
         <property name="text">
          <string>The time between steps in a fade in milliseconds</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="fade_delta"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="fade_in_step_label">
         <property name="text">
          <string>Opacity change between steps while fading in</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="fade_in_step">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.028000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fade_out_step_label">
         <property name="text">
          <string>Opacity change between steps while fading out</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="fade_out_step">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.030000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="no_fading_openclose">
         <property name="text">
          <string>Avoid fade windows in/out when opening/closing</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="backendTab">
      <attribute name="title">
       <string>Backend</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QGroupBox" name="backend">
         <property name="title">
          <string>Rendering backend</string>
         </property>
         <layout class="QVBoxLayout" name="layBackend">
          <item>
           <widget class="QRadioButton" name="backend_xrender">
            <property name="text">
             <string>X Render</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="backend_glx">
            <property name="text">
             <string>GLX (OpenGL)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vspBackend">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>161</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QScrollArea" name="scrAdvanced">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="advancedContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>664</width>
            <height>1095</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QGroupBox" name="gbxGeneral">
             <property name="title">
              <string>General</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QCheckBox" name="detect_client_leader">
                <property name="text">
                 <string>Use WM_CLIENT_LEADER to group windows</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="detect_client_opacity">
                <property name="text">
                 <string>Detect _NET_WM_OPACITY on client windows</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="detect_rounded_corners">
                <property name="text">
                 <string>Try to detect windows with rounded corners</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="detect_transient">
                <property name="text">
                 <string>Use WM_TRANSIENT_FOR to group windows</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="force_win_blend">
                <property name="text">
                 <string>Force all windows to be painted with blending</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="mark_ovredir_focused">
                <property name="text">
                 <string>Mark windows that have no WM frame as active</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="mark_wmwin_focused">
                <property name="text">
                 <string>Try to detect WM windows and mark them as active</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="no_ewmh_fullscreen">
                <property name="text">
                 <string>Do not use EWMH to detect fullscreen windows</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="use_ewmh_active_win">
                <property name="text">
                 <string>Use _NET_WM_ACTIVE_WINDOW to determine which window is focused</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbxBackends">
             <property name="title">
              <string>Backends</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QGroupBox" name="gbxAdvXrender">
                <property name="title">
                 <string>X Render</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QCheckBox" name="xrender_sync_fence">
                   <property name="text">
                    <string>Use X Sync fence to sync clients' draw calls (nvidia drivers)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="gbxGlx">
                <property name="title">
                 <string>GLX</string>
                </property>
                <layout class="QVBoxLayout" name="layGlx">
                 <item>
                  <widget class="QCheckBox" name="glx_no_rebind_pixmap">
                   <property name="text">
                    <string>Avoid rebinding pixmap on window damage</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="glx_no_stencil">
                   <property name="text">
                    <string>Avoid using stencil buffer</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbxDebug">
             <property name="title">
              <string>Debug</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="monitor_repaint">
                <property name="text">
                 <string>Highlight the updated area of the screen. X Render only</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="show_all_xerrors">
                <property name="text">
                 <string>Show all X errors</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbxFade">
             <property name="title">
              <string>Fade</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QCheckBox" name="no_fading_destroyed_argb">
                <property name="text">
                 <string>Do not fade destroyed ARGB windows with WM frame</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbxOpacity">
             <property name="title">
              <string>Opacity</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QCheckBox" name="inactive_opacity_override">
                <property name="text">
                 <string>Inactive opacity set by -i overrides _NET_WM_OPACITY values</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbxOptimize">
             <property name="title">
              <string>Optimize</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QCheckBox" name="vsync">
                <property name="text">
                 <string>Enable/disable VSync</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vsync_use_glfinish">
                <property name="text">
                 <string>Use glFinish() instead of glFlush() for (possibly) better VSync</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="use_damage">
                <property name="text">
                 <string>Enable the use of damage information to help limit the area to paint</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset theme="help-about">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
